bot/util/utilities.py

```diff
@@ -154,24 +154,145 @@ async def cembed(self, ctx, channel: str, *, json_string: str):
         except discord.errors.HTTPException:
             await ctx.send("**__Error:__** Could not parse json. Make sure your last argument is valid JSON.")
         except discord.DiscordException:
-            await ctx.send(
-                "**__Error:__** Invalid embed. Make sure you have at least title or description set (also for each additional field). You can validate your json at https://leovoel.github.io/embed-visualizer/.")
+            await ctx.send("**__Error:__** Invalid embed. Make sure you have at least title or description set (also "
+                           "for each additional field). You can validate your json at "
+                           "https://leovoel.github.io/embed-visualizer/.")
         except ValueError as error:
             await ctx.send(error)
         except TypeError:
             await ctx.send("**__Error:__** Error creating embed. Please check your parameters.")
 
     @commands.command(name="echo")
-    async def echo(self, ctx, channel: typing.Optional[discord.TextChannel], *, text: str):
+    @commands.has_guild_permissions(administrator=True)
+    async def echo(self, ctx: commands.Context, channel: typing.Optional[discord.TextChannel], *, text: str):
         """Lets the bot post a simple message to the mentioned channel (or the current channel if none is mentioned).
 
         Args:
             ctx (discord.ext.commands.Context): The context from which this command is invoked.
             channel (Optional[str]): The channel where the message will be posted in.
-            text (str): The text to be echoed
+            text (str): The text to be echoed.
         """
         await (channel or ctx).send(text)
 
+    @commands.group(name="bot")
+    @commands.has_guild_permissions(administrator=True)
+    async def bot_cmd(self, ctx: commands.Context):
```

Comment:
Wouldn't it make sense, that if you do not use a subcommand it posts infos about the bot? Would make that about command redundant. Or at least put that about command in the bot group. like `bot about` or `bot info`.

Reply:
You're right but I would find it better if a user could simply type `!about` instead of `!bot about`. I'm gonna try to create an alias for this.